backend/blueprint/api/admin/users.py

```diff
@@ -0,0 +1,137 @@
+from flask import Blueprint, jsonify, request
+from api.utils.response import Response
+from firebase_admin import auth
+
+
+router = Blueprint('users', __name__)
+
+
+@router.route('/createuser', methods=['POST'])
+def create_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+        
+    try:
+        user = auth.create_user(
+            display_name=data['name'],
+            email=data['email'],
+            email_verified = data['email_verified'],
+            photo_url=data['photo_url'],
+            password=data['password'],
+            disabled = data['disabled']
+        )
+        return jsonify({
```

Comment:
Try to use `Response` class that is already present. That is already present and acts as a convention.

Reply:
It supports string that can be passed as argument??